faster-map.opam

```diff
@@ -1,9 +1,14 @@
-opam-version: "1.2"
+opam-version: "2.0"
+synopsis: "A tail-recursive list map with good performance for all list sizes. Not actually written in assembly"
+authors: "<Anton Bachin> antonbachin@yahoo.com"
```

Comment:
IIRC this is should be `name <email>`, not `<name> email`.

Reply:
Thanks.